cpp/dolfinx/fem/Function.h

```diff
@@ -153,28 +153,33 @@ class Function
   /// @brief Interpolate a provided Function.
   /// @param[in] v The function to be interpolated
   /// @param[in] cells The cells to interpolate on
+  /// @param[in] cell_map Map from indices in `cells` to cell indices in `v`
```

Comment:
Maybe this is clearer: `cell_map` A map from cells in the mesh associated with `this` function to cells in mesh associated with `v`